projects/VitalRecord/VitalRecord_util.cs

```diff
@@ -1111,6 +1112,37 @@ protected void SetCodeValue(string field, string code, string[,] options)
             throw new System.ArgumentException($"Code '{code}' is not an allowed value for field {field}");
         }
 
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="code"></param>
+        /// <param name="options"></param>
+        /// <exception cref="System.ArgumentException"></exception>
+        protected Dictionary<string, string> CreateCode(string code, string[,] options)
```

Comment:
I think this code block exists in a couple places, is it possible to replace other occurrences of this code block with CreateCode?